css/tools/apiclient/test.py

```diff
@@ -11,6 +11,7 @@
 #  [1] http://www.w3.org/Consortium/Legal/2002/copyright-software-20021231
```

Comment:
It's not clear to me if we want to keep this in sync with upstream; CC @gsnedders @plinss

Reply:
Yeah, let's not fork this.

Reply:
What is the repo for upstream.  These are _syntax errors_ that crash will the runtime.

Reply:
Given the CSS test suite build system has been essentially unmaintained for years and it seems unlikely anyone is going to put in the work to get it running on Py3 (and the single digit number of users will likely just rely on whatever RH fork of Py2 exists in the near-term), I'd just not bother. There's much more than just syntax errors in that codebase and having it fail fast at least makes it clear it's unsupported.

You can try submitting a PR to https://github.com/w3c/csswg-apiclient and try and get @plinss to merge it upstream, but I'd suggest it's not worth the effort.

Reply:
https://github.com/w3c/csswg-apiclient/pull/2